api.rst

```diff
@@ -670,7 +670,7 @@ As an example, let's create a view called ``nominations_view`` based on the *nom
   FROM
     nominations;
 
-Since it contains ``competition_id`` and ``film_id``—and each one has a **foreign key** defined in its source table—we can embed *competitions* and *films*:
+Since it contains ``competition_id`` and ``film_id`` — and each one has a **foreign key** defined in its source table — we can embed *competitions* and *films*:
```

Comment:
The main reason for this change was, because it broke my vim-syntax highlighting for the remaining file.. but I think it's better to read as well.